imports/plugins/custom/payments-wallet/client/dashboard/components/WalletAction.js

```diff
@@ -65,6 +89,22 @@ class WalletAction extends Component {
                 />
               </label>
 
+              {
+                actionType === "transfer"
+                &&
```

Comment:
You could put this on the same line with `actionType === "transfer"`

Reply:
@chinezenwosu I don't understand what you meant by You could put this on the same line with actionType === "transfer"